config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+fT7N+Lzx6KQl2EfG/PC+L8tvvg3wxw35ehpH8opveAzSS0sRJ4sCghqtgp6I7+G7Jin0RXjhPxBdKXFFl1gMAjv+HvUXlvwbmYhApvUnu+7MQTZJtjN5n2zfMB89+IGcxZYOpzbb7buTafClaWoIJIez+AVdx5tC5e5Jv2vDlq8ZwQtCpVghu0s5aebCJIsEscQ5j6RSz5LiaOKepJM8HBMPt4whfc41/c+/NCUJgIALgZv2zyrIdmo+Q5PGjROqHmLzM79p7GaCg7d1AV585kTO05ovYkfBIVCQ6f+AevBLSSUcbqTHH9jIKYHbvqnFg55mtEMikCGgx3XUtXKMIc8r9eiAqcFoCWYS8ebmuXqTuwiVhB3qjy9i19y1CeeW+EOFZxNf0Fo6dyeXrYlcLBmk4AZvGAe4VJa4--vn7yBSIuc4+0JVoB--+24yLLFWCQT+Grm97uXnzQ==
```

Comment:
Take care when using the encrypted credentials

Are you going to use them or was also autogenerated?

Reply:
Same as before, It was auto-generated